src-opam/dockerfile_distro.ml

```diff
@@ -109,14 +109,14 @@ let distro_status (d:t) : status = match d with
   | `Ubuntu ( `V12_04 | `V14_04 | `V15_04 | `V15_10 | `V16_04 | `V16_10 | `V17_04 | `V17_10 | `V18_10 | `V19_04 | `V19_10 ) -> `Deprecated
   | `Ubuntu `LTS -> `Alias (`Ubuntu `V20_04)
   | `Ubuntu `Latest -> `Alias (`Ubuntu `V20_10)
-  | `Cygwin `V20H2 -> `Active `Tier3
+  | `Cygwin `V21H1 -> `Active `Tier3
   | `Windows (_, _) -> `Active `Tier3
 
 let latest_distros =
   [ `Alpine `Latest; `Archlinux `Latest; `CentOS `Latest;
     `Debian `Stable; `OracleLinux `Latest; `OpenSUSE `Latest;
     `Fedora `Latest; `Ubuntu `Latest; `Ubuntu `LTS;
-    `Cygwin `V20H2; `Windows (`Mingw, `V20H2); `Windows (`Msvc, `V20H2)]
+    `Cygwin `V21H1; `Windows (`Mingw, `V21H1); `Windows (`Msvc, `V21H1)]
```

Comment:
This is the tag we use for the unmarked version? I think policy-wise it might be worth lagging one month behind release - i.e. allow the first LCU update. Equally, if there's a problem we can rebuild the docker image, so I'm not certain about that!